src/ol_infrastructure/infrastructure/aws/network/Pulumi.infrastructure.aws.network.CI.yaml

```diff
@@ -5,6 +5,8 @@ config:
   apps_vpc:cidr_block: 172.18.0.0/16
   aws:region: us-east-1
   data_vpc:cidr_block: 172.17.0.0/16
+  k8s_vpc:cidr_block: 172.30.0.0/16
+  k8s_vpc:k8s_service_subnet: 172.30.48.0/20
```

Comment:
Are you creating the additional VPC just for testing and to have a place for experimentation, or is your thinking that we'll be using it for our actual K8s workloads?

Reply:
Wait did I miss something? I thought yesterday we decided the best course was to give K8S its own VPC and peer it as needed? (no peering here, yet, though...)

Reply:
No, you didn't miss anything. I know we had discussed it but wasn't sure what the final determination was. I'm happy to go either route, we'll just need to consider what any migration paths might look like as we consolidate into the one cluster/VPC. Specifically, any reassociation of RDS, Elasticache, and OpenSearch resources into the new VPC.

Reply:
OK. I think this change leaves both paths viable and I will keep it in the front of my mind as I write the rest of the EKS stuff.